esignet-core/src/main/java/io/mosip/esignet/core/exception/EsignetException.java

```diff
@@ -6,11 +6,18 @@
 package io.mosip.esignet.core.exception;
 
 import io.mosip.esignet.core.constants.ErrorConstants;
+import lombok.Getter;
+import lombok.Setter;
 
 public class EsignetException extends RuntimeException {
 
+    @Getter
     private String errorCode;
 
+    @Setter
+    @Getter
+    private String dpopNonceHeaderValue;
```

Comment:
It will be a clean approach to create a separate exception class "DPoPNonceMissingException"